packages/scripts/scripts/test-unit-jest.js

```diff
@@ -18,22 +18,15 @@ const jest = require( 'jest' );
  * Internal dependencies
```

Comment:
Why do we have a `test-unit-js` that simply points to `test-unit-jest`?

Reply:
In core they use QUnit to run unit tests, they wanted to have Jest being exposed as one of the options and default unit tests to Jest.

We don't have an alternative for e2e testing, so I decided to skip the linking step.